terraform/environments/analytical-platform-ingestion/dms/kms-keys.tf

```diff
@@ -13,3 +13,15 @@ module "s3_cica_dms_ingress_kms" {
 
   deletion_window_in_days = 7
 }
+
+module "cica_dms_credentials_kms" {
```

Comment:
Can you add `tags = local.tags` to this module please

Reply:
Done